opentelemetry-proto/src/proto.rs

```diff
@@ -1,66 +1,68 @@
 #[cfg(feature = "gen-tonic")]
+#[path = "proto/tonic"]
```

Comment:
I think we should avoid generating `#[path]` attributes for modules that are defined here (as opposed to just being declared).

Reply:
Not sure I understand. Are you proposing we generate the folder structure for generated files?

Reply:
I'm saying we should not generate `#[path = ""]` attributes.

Reply:
I think if we don't annotate those mods with `#[path = ""]` then compiler will try to add another directory with the same name as the module when searching for files. 

For example, `tonic::log::v1` will try to find the file in `proto/tonic/log/v1` instead of `proto/tonic`. Not sure if there is a better way to do it.

Reply:
Well, the better way to do it is to write out the module hierarchy to the filesystem, as we do in stackdriver, IMO. But I guess this is fine too if we actually need those empty-`path` attributes.